core-services/prow/02_config/_config.yaml

```diff
@@ -314,15 +314,6 @@ slack_reporter_configs:
     channel: '#ops-testplatform'
     report_template: Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}.
       <{{.Status.URL}}|View logs>
-  openshift-assisted/assisted-installer-deployment:
```

Comment:
weird to see this change in your PR (I did it a couple of days ago)
it only affect the ownership, making dptp required to approve it. so I suggest you to rebase with master and remove this change from your commit